config/config.json.in

```diff
@@ -1,7 +1,6 @@
 {
   "base": {
-    "xlen": 64,
-    "E": false,
+    "xlen": @CONFIG__BASE__XLEN@,
```

Comment:
Any reason you dropped the E entry here?

Reply:
See #1228.